pkg/printers/internalversion/describe.go

```diff
@@ -1264,6 +1264,20 @@ func describePersistentVolumeClaim(pvc *api.PersistentVolumeClaim, events *api.E
 		if pvc.Spec.VolumeMode != nil {
 			w.Write(LEVEL_0, "VolumeMode:\t%v\n", *pvc.Spec.VolumeMode)
 		}
+		if len(pvc.Status.Conditions) > 0 {
+			w.Write(LEVEL_0, "Conditions:\n")
+			w.Write(LEVEL_1, "Type\tStatus\tLastProbeTime\tLastTransitionTime\tReason\tMessage\n")
+			w.Write(LEVEL_1, "----\t------\t-----------------\t------------------\t------\t-------\n")
+			for _, c := range pvc.Status.Conditions {
+				w.Write(LEVEL_1, "%v \t%v \t%s \t%s \t%v \t%v\n",
+					c.Type,
+					c.Status,
+					c.LastProbeTime.Time.Format(time.RFC1123Z),
```

Comment:
Have you verified the ouput manually? We don't really set `LastProbeTime` currently but a future version of k8s may start setting it, so it makes sense to display it but we don't want to display incorrect value.

Reply:
I set `LastProbeTime` to `time.Now()` manually and the output is as below, It looks right:
```
Conditions:
  Type                      Status  LastProbeTime                     LastTransitionTime                Reason  Message
  ----                      ------  -----------------                 ------------------                ------  -------
  FileSystemResizePending   True    Tue, 06 Mar 2018 10:54:09 +0800   Tue, 06 Mar 2018 10:53:59 +0800           Waiting for user to (re-)start a pod to finish file system resize of volume on node.

```